src/client/v2/indexer/searchForApplicationBoxes.ts

```diff
@@ -0,0 +1,134 @@
+import JSONRequest from '../jsonrequest';
+import HTTPClient from '../../client';
+import IntDecoding from '../../../types/intDecoding';
+import { BoxDescriptor } from '../algod/models/types';
+
+export interface SearchForApplicationBoxesResponse {
```

Comment:
I'm leaving this comment as a visual reminder to look into indexer response models more.

Right now this SDK only uses algod's REST API definitions to generate the models, which means some indexer-specific models like this one are missing. Ideally we would be able to generate these models as well instead of manually creating them here.

Reply:
I opened #662 which addresses this